buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -492,7 +492,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def postgres_version = "42.2.16"
     def powermock_version = "2.0.9"
     // Try to keep protobuf_version consistent with the protobuf version in google_cloud_platform_libraries_bom
-    def protobuf_version = "3.21.12"
+    def protobuf_version = "3.22.1"
```

Comment:
https://github.com/googleapis/java-cloud-bom/releases/tag/v26.10.0

> Protobuf Java: 3.21.12

Don’t update this line. The latest Protobuf Java is known to have invalid pom.xml file.

This issue https://github.com/protocolbuffers/protobuf/issues/12170

Reply:
is it ok to leave protobuf_version out of sync with the BOM then?

Reply:
I mean keep in sync with the BOM. This change is making out of sync.

Reply:
As far as I can tell from here https://mvnrepository.com/artifact/com.google.cloud/libraries-bom/26.10.0 the 26.10.0 BOM is using protobuf 3.22.1 is that not right?

Reply:
I don't use mvnrepository.com. The official site is https://central.sonatype.com.

https://central.sonatype.com/artifact/com.google.cloud/libraries-bom/26.10.0 uses https://central.sonatype.com/artifact/com.google.cloud/first-party-dependencies/3.4.0, which uses https://central.sonatype.com/artifact/com.google.api/gapic-generator-java-bom/2.15.2, which uses https://central.sonatype.com/artifact/com.google.api/gapic-generator-java-pom-parent/2.15.2, which sets `<protobuf.version>3.21.12</protobuf.version>`.

To avoid the manual work, the release note https://github.com/googleapis/java-cloud-bom/releases/tag/v26.10.0 shows

> Protobuf Java: 3.21.12

Reply:
fixed, thank you.